src/assets/server/rest/v1/schemas/user/users-get.json

```diff
@@ -6,10 +6,6 @@
       "type": "boolean",
       "sanitize": "mongo"
     },
-    "WithTag": {
-      "type": "boolean",
-      "sanitize": "mongo"
-    },
```

Comment:
No need for withTag flag anymore as we don't handle the "import users with tags"
@LucasBrazi06 can you confirm please ?